specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/codeVersion.json

```diff
@@ -0,0 +1,400 @@
+          "type": "boolean"
+        }
+      },
+      "additionalProperties": false
+    },
+    "CodeVersion": {
+      "description": "Details of an RegisteredCode",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The Name of the Asset.",
+          "type": "string",
+          "readOnly": true
+        },
+        "version": {
+          "description": "The Asset version assigned by service.",
+          "type": "string",
+          "readOnly": true
+        },
+        "description": {
+          "description": "The Asset description text.",
+          "type": "string"
+        },
+        "datastoreId": {
+          "description": "The Asset datastoreId",
+          "type": "string"
+        },
+        "tags": {
+          "description": "The Asset tag dictionary.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "description": "The Asset property dictionary.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "assetPaths": {
+          "description": "Details of asset path\r\nAdding list of back compat",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssetPath"
+          }
+        },
+        "linkedResourceIds": {
+          "description": "Associated ARM resources. Key is the link type, value is a list of ARM IDs.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "CodeVersionResource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The resource URL of the entity (not URL encoded).",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the resource entity.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The resource provider and type.",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/CodeVersion"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "additionalProperties": false,
+      "description": "Azure Resource Manager resource Envelope"
+    },
+    "CodeVersionResourcePaginatedResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CodeVersionResource"
+          }
+        },
+        "continuationToken": {
```

Comment:
the collection GET contract should only contain "value" and "nextLink"

Reply:
fixed